### PR TITLE
Consolidate duplicate files

### LINKED ISSUE #23905: Consolidate with duplicate article
This article should be consolidated with https://docs.microsoft.com/en-us/dotnet/standard/language-independence-and-language-independent-components. They are almost identical.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bb9b4b39-813f-ed23-3cb7-682766b3cfaa
* Version Independent ID: c3216a96-76a2-21aa-2072-0328725cd3cb
* Content: [Language independence and language-independent components](https://docs.microsoft.com/en-us/dotnet/standard/language-independence)
* Content Source: [docs/standard/language-independence.md](https://github.com/dotnet/docs/blob/main/docs/standard/language-independence.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @gewarren
* Microsoft Alias: **gewarren**

### PR DESCRIPTION
**Pull request by gewarren**
Fixes #23905.

In the longer-named file that I'm deleting, all the code snippets are in separate files instead of inline. Is that preferred? Otherwise we should delete the snippet files too.

### CODE REVIEW

**Comment by tdykstra** [comment]
>In the longer-named file that I'm deleting, all the code snippets are in separate files instead of inline. Is that preferred? Otherwise we should delete the snippet files too.
Separate files is preferred, especially if they have an accompanying .csproj file so they get validated. That said, if making them compilable is too much work, leaving them inline for now might be the best choice.

File: docs/standard/language-independence.md [hunk 1]
```diff
@@ -26,77 +25,20 @@ This section discusses the rules for creating a CLS-compliant component. For a c
...
-You can make the Person class CLS-compliant by changing the type of `Age` property from `UInt16` to [Int16](xref:System.Int16), which is a CLS-compliant, 16-bit signed integer. You do not have to change the type of the private `personAge` field.
+For example, unsigned integers other than <xref:System.Byte> are not CLS-compliant. Because the `Person` class in the following example exposes an `Age` property of type <xref:System.UInt16>, the following code displays a compiler warning.
 
-```csharp
-using System;
+[!code-csharp[Conceptual.CLSCompliant#1](../../samples/snippets/csharp/VS_Snippets_CLR/conceptual.clscompliant/cs/public1.cs#1)]
+[!code-vb[Conceptual.CLSCompliant#1](../../samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.clscompliant/vb/public1.vb#1)]
 
-[assembly: CLSCompliant(true)]
+You can make the `Person` class CLS-compliant by changing the type of `Age` property from <xref:System.UInt16> to <xref:System.Int16>, which is a CLS-compliant, 16-bit signed integer. You do not have to change the type of the private `personAge` field.
```
Comment (tdykstra) [inline docs/standard/language-independence.md:38]:
```suggestion
You can make the `Person` class CLS-compliant by changing the type of the `Age` property from <xref:System.UInt16> to <xref:System.Int16>, which is a CLS-compliant, 16-bit signed integer. You don't have to change the type of the private `personAge` field.
```

File: docs/standard/language-independence.md [hunk 2]
```diff
@@ -314,1591 +160,197 @@ Non-compliant type | Description | CLS-compliant alternative
...
-  public unsafe class TestClass
-  {
-     private int* val;
+If a type is not CLS-compliant, you should apply the <xref:System.CLSCompliantAttribute> attribute with an `isCompliant` value of `false` to it. For more information, see [The CLSCompliantAttribute attribute](#the-clscompliantattribute-attribute) section.
 
-     public TestClass(int number)
-     {
-        val = (int*) number;
-     }
+The following example illustrates the problem of CLS compliance in a method signature and in generic type instantiation. It defines an `InvoiceItem` class with a property of type <xref:System.UInt32>, a property of type `Nullable(Of UInt32)`, and a constructor with parameters of type <xref:System.UInt32> and `Nullable(Of UInt32)`. You get four compiler warnings when you try to compile this example.
```
Comment (tdykstra) [inline docs/standard/language-independence.md:173]: Is there a reason for using VB syntax here?

**Review by tdykstra — APPROVED** [review]
LGTM

File: docs/standard/language-independence.md (hunk 2 above)
Comment (gewarren) [reply docs/standard/language-independence.md]: Good catch, not sure why it was written that way.

**Comment by gewarren** [comment]
Thanks for the review Tom!

**Pull request merged**